module/assessments/helper.js

```diff
@@ -575,6 +578,11 @@ module.exports = class AssessmentsHelper {
                 entities : requestedData.entities ? requestedData.entities: []
               };
 
+              if( requestedData.taskId && requestedData.projectId ) {
```

Comment:
@aman-tunerlabs - Not required inside solutionInformation as already present outside.